frontend/main/components/NavigationMenu/NavigationMenu.js

```diff
@@ -36,10 +36,10 @@ export class NavigationMenu extends BaseComponent {
 
 
     const links = [
-      { text: "Marketplace", src: "../../../assets/navigation-menu/sell-0.svg", className: "marketplace-button", target: "marketplace" },
-      { text: "Seller Page", src: "../../../assets/navigation-menu/save-0.svg", className: "sell-button", target: "sellProductsPage" },
-      { text: "Virtual Cart", src: "../../../assets/navigation-menu/cart-0.svg", className: "cart-button", target: "virtualCart" },
-      { text: "User Center", src: "../../../assets/navigation-menu/profile-0.svg", className: "profile-button", target: "profilePage" },
+      { text: "Marketplace", iconName: "fa-shop", className: "marketplace-button", target: "marketplace" },
```

Comment:
Icons looks good! They looks similar to the original design but are better with filled status. Thank you for looking at other sources and enhancing the consistency for nav-menu Benson!

Reply:
Just in case, can you find the orange icons as well? I may want to enhance the menu by adding the focus state with orange text and icons. Thank you!

Reply:
color icon costs money, unless someone can copy the icon and make a custom icon that's colored orange using something like canva, etc.

Reply:
I see, let me find some other ways to show status. Thank you for info!